derp/derp_server.go

```diff
@@ -774,10 +773,11 @@ func (c *sclient) handleFrameClosePeer(ft frameType, fl uint32) error {
 	if !c.canMesh {
 		return fmt.Errorf("insufficient permissions")
 	}
-	var targetKey key.Public
-	if _, err := io.ReadFull(c.br, targetKey[:]); err != nil {
+	var bs [32]byte
+	if _, err := io.ReadFull(c.br, bs[:]); err != nil {
 		return err
 	}
+	targetKey := key.NodePublicFromRaw32(mem.B(bs[:]))
```

Comment:
Why do a ReadFull into an untyped buffer and then do a FromRaw32? Can we instead do a single key.ReadFrom call? (I have already forgotten the exact name.) And then maybe we get delete FromRaw32?

Reply:
I was doing a mechanical change and didn't notice this pattern. Fixed, but there are still some FromRaw32 calls in derp_client that aren't as obviously deletable yet. Closer to a tight API though...